de_workloads/data_processing/silver/silver.yml

```diff
@@ -163,7 +163,7 @@ stages:
                 - task: AzureCLI@2
                   inputs:
                     # TODO: Remove hardcoded reference to subscription
```

Comment:
please remove the todo comments. Also, the same change is now required in .jinja templates (de_templates)

Reply:
I removed the TODO comments from pipelines. I can the see .jinja templets doe not have latest changes as well.

Reply:
thank you, as discussed: if not too much work, please add the missing changes to the templates as well.